lib/Runtime/Language/AsmJsCodeGenerator.cpp

```diff
@@ -37,6 +37,10 @@ namespace Js
 #if ENABLE_DEBUG_CONFIG_OPTIONS
             funcEntrypointInfo->SetIsTJMode(true);
 #endif
+            if (!PreReservedVirtualAllocWrapper::IsInRange((void*)mScriptContext->GetThreadContext()->m_prereservedRegionAddr, (void*)address))
+            {
+                mScriptContext->GetJitFuncRangeCache()->AddFuncRange((void*)address, (int)entrypointInfo->GetCodeSize());
```

Comment:
Checked the upper bound for uint 32bit

---
In reply to: [100214944](https://github.com/Microsoft/ChakraCore/pull/2496#discussion_r100214944) [](ancestors = 100214944)